package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "underover-districts",
+  "name": "undercover-districts",
```

Comment:
Not sure, but I thought this was a typo.

Reply:
Ha, it wasn't a typo: it was `underover` as in **Under**population, **over**population. But I see why you thought it was.

I will rename this to match the repo name: `us-cd-population-map`. I think that makes more sense. `underover` was a placeholder from early in the project.